package.json

```diff
@@ -7,7 +7,7 @@
         "prepare": "husky install",
         "test": "ts-mocha src/tests/**.ts",
         "dev": "yarn && ts-node-dev --transpile-only --no-notify --exit-child src/index.ts",
-        "generate": "yarn && graphql-codegen --config ./src/typesgeneratorconfig.ts",
```

Comment:
Let's not call yarn inside yarn here, it tries to install husky in docker 😭

Reply:
haha 😅